### PR TITLE
[CI-21] `get_quizzes` endpoint

### PR DESCRIPTION
**Pull request by rcybulski1122012**
I've forked the branch from `feat/CI-20` to not wait for the code review, so I temporarily changed the base branch.

### CODE REVIEW

File: backend/app/services/quiz.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from typing import Sequence
+
+from sqlalchemy import select
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.models.quiz import Quiz
+
+
+async def get_quizzes(session: AsyncSession) -> Sequence[Quiz]:
```
Comment (rcybulski1122012) [inline backend/app/services/quiz.py:9]: I thought about creating a repository or DAO, but for this size of project it would be an overkill

Comment (BARBUR02) [reply backend/app/services/quiz.py:9]: Yeah i agree 100%

File: backend/app/services/quiz.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    statement = select(Quiz).order_by(Quiz.created_at)
+    result = await session.execute(statement)
+    return result.scalars().all()
```
Comment (BARBUR02) [inline backend/app/services/quiz.py:12]: There is actually a nice trick to use:
```suggestion
    return await session.scalars(statement).all()
```
(if i'm not mistaken 😄 )

**Review by BARBUR02 — APPROVED** [review]
Good job!

**Comment by rcybulski1122012** [comment]
I've just realized that I forgot about pagination. Do we want it? What kind?

**Comment by BARBUR02** [comment]
I don't think we need that for available quizzes list.

File: backend/app/services/quiz.py (hunk 2 above)
Comment (rcybulski1122012) [reply backend/app/services/quiz.py:12]: `.scalars` is a coroutine so we'd need to do
```python
    return (await session.scalars(statement)).all()
```
which in my opinion is less readable

Comment (IwoSzczepaniak) [reply backend/app/services/quiz.py:12]: I agree with Radek

**Review by IwoSzczepaniak — APPROVED** [review]
Ok!

**Pull request merged**